src/modules/comment/services/comment.service.spec.ts

```diff
@@ -15,6 +15,15 @@ describe('CommentService', () => {
   let commentService: CommentService;
   let prismaService;
   const commentModels: PrismaCommentModelName[] = ['lessonComment'];
+  let createCommentColumn = (
```

Comment:
```suggestion
  const createCommentColumn = (
```
const 로 선언되면 문제가 생기나요???

Reply:
아뇨 아래에 있던 코드를 위로 올리면서 const로 변경해주질 않았을 뿐입니다 ㅎ 변경하겠슴다